src/OpenTelemetry.Contrib.Exporter.ApplicationInsights/OpenTelemetry.Contrib.Exporter.ApplicationInsights.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Contrib.Extensions.AWSXRay/OpenTelemetry.Contrib.Extensions.AWSXRay.csproj

Model this similar to the AWSExtension. This is *not* an exporter, but something like an extension. So it feel confusing to call it "Exporter"